docs/modules/ROOT/pages/migration.adoc

```diff
@@ -479,6 +479,13 @@ class WebSocketSecurityConfig: WebSocketMessageBrokerConfigurer {
 ----
 ====
 
+[[servlet-opt-in-sha256-rememberme]]
+==== Use SHA-256 in Remember Me
+
+The `TokenBasedRememberMeServices` implementation now supports SHA-256 for the Remember Me token and this is the default in Spring Security 6.
```

Comment:
I think it would be helpful to tell the reader how to declaratively maintain the 5.8 default, just in case moving to SHA-256 breaks things.